docs/basics/scenemanagement/custom-management.md

```diff
@@ -10,32 +10,43 @@ Custom Scene Management currently has some drawbacks that we hope to improve upo
+
+### Integrated NetworkSceneManager Comparison (Enabled vs Disabled)
+Network Scene Management      | Enabled  | Disabled
+------------------------------|----------|----------
+Scene Event Synchronizing     | Yes      | No
+In-Scene Network Prefab Synch | Yes      | Yes
+In-Scene NetworkObject Synch  | Yes      | No
+Auto scene migration Synch    | Yes      | No
+
+#### Scene Event Synchronizing
+When integrated scene management is enabled, scene related events are automatically synchronized for you. When integrated scene management is disabled, you need to handle the synchronization of scenes yourself.
+
+#### In-Scene Placed Network Prefabs and NetworkObjects
+Both when enabled and disabled, a joining client is synchronized with any in-scene placed Network Prefab instances. When integrated scene management is disabled, only the scenes loaded prior to a client joining will have their in-scene placed Network Prefab instance synchronized. If you load any other scenes after a client is connected, they will not be synchronized (you would want to only dynamically spawn NetworkObjects at that point).
+
+:::warning Any in-scene defined NetworkObject (i.e. not a network prefab instance) will not get synchronized with a newly joined client when scene management is disabled.
 :::
 
-Once you have registered your in-scene placed Network Prefab instances with the `NetworkManager`, you can then start a server/host and have a client connect and synchronize with the in-scene placed Network Prefab instances (as long as both client and server have pre-loaded the scene or scenes required).  Due to the complexity of in-scene placed Network Prefab registration, we highly recommend using in-scene placed `NetworkObject`s sparingly and [use the Hybrid approach](inscene-placed-networkobjects#a-hybrid-approach-example) as much as possible. 
+When the integrated `NetworkSceneManager` is enabled both in-scene placed network prefab instances and in-scene defined `NetworkObject`s get synchronized and when new scenes are loaded they get synchronized.
+
+#### Auto (NetworkObject) Scene migration Synchronization
+When integrated scene management is enabled, you can have the server migrate an already spawned `NetworkObject` from one scene to another and it will automatically be synchronized with connected and late joining clients.
+When integrated scene management is disabled, this is not handled and you will need to use either NetworkVariables, Rpcs, or custom messages to handle synchronizing clients with any migration of `NetworkObjects`s from one scene to another.
```

Comment:
```suggestion
#### Auto (NetworkObject) Scene Migration Synchronization
When integrated scene management is enabled, you can have the server migrate an already spawned `NetworkObject` from one scene to another and it will automatically be synchronized with connected and late-joining clients.

When integrated scene management is disabled, this is not handled and you need to use either NetworkVariables, Rpcs, or custom messages to handle synchronizing clients with any migration of `NetworkObjects`s from one scene to another.
```